tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableTest.java

```diff
@@ -1427,6 +1427,18 @@ public void testRemoveOp() {
         Assert.assertEquals(returns[0].stringValue(), "table<Order> {index: [], primaryKey: [], data: []}");
     }
 
+    @Test(groups = TABLE_TEST, description = "Check inserting byte array data as direct parameter.")
+    public void testByteArrayInsertAsDirectParameter() {
+        BValue[] returns = BRunUtil.invoke(result, "testByteArrayInsertAsDirectParameter");
+        Assert.assertEquals(((BInteger) returns[0]).intValue(), 1);
+    }
+
+    @Test(groups = TABLE_TEST, description = "Check inserting byte array data as SqlParameter.")
+    public void testByteArrayInsertAsSqlParameter() {
+        BValue[] returns = BRunUtil.invoke(result, "testByteArrayInsertAsSqlParameter");
+        Assert.assertEquals(((BInteger) returns[0]).intValue(), 1);
+    }
+
```

Comment:
Is there a possibility to make these two tests a part of an existing test? Do we have a specific reason to make these separate tests?

Reply:
Moved testByteArrayInsertAsSqlParameter test case to an existing test case. 
Changed other test to do the sql operation in the existing db but keeping it as separate test, since there are no test case written by passing params as direct params .

Reply:
Moved to part of an existing test.
https://github.com/ballerina-platform/ballerina-lang/pull/15410/commits/4c5751d7c86e5153c50efc46c775120e159d4101